include/ddc/kokkos_allocator.hpp

```diff
@@ -50,6 +51,11 @@ class KokkosAllocator
         return static_cast<T*>(Kokkos::kokkos_malloc<MemorySpace>(sizeof(T) * n));
     }
 
+    [[nodiscard]] T* allocate(std::string const& label, std::size_t n) const
+    {
+        return static_cast<T*>(Kokkos::kokkos_malloc<MemorySpace>(label, sizeof(T) * n));
```

Comment:
shouldn't we use the lower-level `MemorySpace::allocate`?

Reply:
Maybe in the end. What would be the decision criterion ? The fact that we don't take advantage of the memory tracking ?

Reply:
exactly

Reply:
ok :D let's go with kokkos_malloc then!